docs/source/notebooks/ogb_biokg_demo.ipynb

```diff
@@ -31,13 +31,13 @@
    "source": [
     "import sys\n",
     "!{sys.executable} -m pip uninstall -y kg_topology_toolbox\n",
-    "!pip install -q git+https://github.com/graphcore-research/kg-topology-toolbox.git\n",
+    "!pip install -q git+https://github.com/graphcore-research/kg-topology-toolbox.git@refactor_kgtt --no-cache-dir\n",
```

Comment:
I guess we will need to remember to remove the @refactor_kgtt once we merge to main!

Reply:
Good spot! I removed this right away, after testing the notebook, so that we don't forget later